instrumentation/opentelemetry-instrumentation-boto/test-requirements.txt

```diff
@@ -5,7 +5,7 @@ botocore==1.34.44
 certifi==2024.2.2
 cffi==1.16.0
 charset-normalizer==3.3.2
-cryptography==42.0.3
+cryptography==42.0.4
```

Comment:
nit: I noticed we are using `cryptography==42.0.5` in Botocore. Does it make sense to pin this version here, too?

Reply:
sure